### ISSUE TITLE
into module?

### ISSUE THREAD

**Issue by MateuszKSS** [NordicSemiconductor/nRF51-ble-bcast-mesh#181]
Is it possible to implement this solution into module based on nRF52 chip?

**Comment by trond-snekvik** [comment]
Yes, but for now, we only support a Keil build for the nRF52. You can build for nRF52 by linking the SDK12 (clone the repo into your SDK12 examples-folder instead of SDK8), and select the nRF52832 xxAA target for the BLE gateway example.

**Comment by thedjnK** [comment]
Can confirm (with edits) you can get a gcc build working fine too
